core/arch/arm/plat-sam/main.c

```diff
@@ -44,6 +44,451 @@
 #include <sm/optee_smc.h>
 #include <tz_matrix.h>
 
+#define MATRIX_AXIMX   1
+#define MATRIX_H64MX   2
+#define MATRIX_H32MX   3
+
+static struct matrix matriies[] = {
+	{.matrix = MATRIX_H64MX, .base_addr = AT91C_BASE_MATRIX64, .va = 0,},
+	{.matrix = MATRIX_H32MX, .base_addr = AT91C_BASE_MATRIX32, .va = 0,},
```

Comment:
No need to init fields initialized to 0.
Add a space char after/before opening/closing braces.